tests/functional/wdio.conf.js

```diff
@@ -45,14 +45,15 @@ export const config = {
   // and 30 processes will get spawned. The property handles how many capabilities
   // from the same test should run tests.
   //
-  maxInstances: 2,
+  maxInstances: 1,
```

Comment:
Did we have to drop this?.

Reply:
My change didnt save earlier when adding the commit. Just pushed before ur comment 😂

Reply:
I'm not following, did it need to be lowered?.

Reply:
No, 1 Max instance works fine. Unless we wanted to keep 2 ? Was just saying the instance change didnt get pushed when I initially git added

Reply:
Yes, it should be 2 unless it can't be. Lowering it to 1 is going to twice as long to run.

Reply:
Oh I understand now, reverted. Will have a look at the actions results again

Reply:
Reverting back to 1 instance because tests are failing too frequently, needing to retry multiple times

Reply:
In that case, I would set this to use an env var and default it to 2 so locally it isn't 1. Then in Actions it should override to use 1. Similar to how we do `EQ_RUN_FUNCTIONAL_TESTS_HEADLESS`.

Reply:
Also, why does it fail at 2, is it to due with the issue I linked previously or some other reason?. Is it something that we can't get around?.

Reply:
Had a look for a while now cannot seem to find a reason. Will revert back to two because it is working locally fine.